packages/devtools_app/test/flutter/vm_flag_widgets_test.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/profiler/profile_granularity.dart';
+import 'package:devtools_app/src/ui/fake_flutter/_real_flutter.dart';
+import 'package:devtools_app/src/ui/flutter/vm_flag_widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'wrappers.dart';
+
+void main() {
+  group('Profile Granularity Dropdown', () {
+    testWidgets('displays with default content', (WidgetTester tester) async {
```

Comment:
If you want to be a little more sure of the dropdown, you can also try a test that taps the dropdownkey, then the selected display text to change the value.

```dart
await tester.tap(find.byKey(...));
await tester.pumpAndSettle();
await tester.tap(find.text(low.display));
await tester.pumpAndSettle();
expect(value, low.value);
```

Reply:
added test for selection